src/lxc/criu.c

```diff
@@ -256,6 +260,10 @@ static void exec_criu(struct criu_opts *opts)
 		if (ret < 0 || ret >= sizeof(log))
 			goto err;
 
+		if (criu_version >= CRIU_IN_FLIGHT_SUPPORT) {
+			DECLARE_ARG("--skip-in-flight");
```

Comment:
I guess adding it to migrate_opts also means that it will be a command-line option to lxc-checkpoint, right?